integration/tests/command_line/negative/neg_invalid_cli_env_flag.rb

```diff
@@ -0,0 +1,23 @@
+require 'git_utils'
+require 'r10k_utils'
+require 'master_manipulator'
+test_name 'CODEMGMT-90 - C63462 - Specify Invalid Command-line Environment Flag'
+
+#Init
+master_certname = on(master, puppet('config', 'print', 'certname')).stdout.rstrip
+git_environments_path = '/root/environments'
+last_commit = git_last_commit(master, git_environments_path)
+
+#Verification
+error_message_regex = /environment: illegal option/
+
+#Teardown
+teardown do
+  clean_up_r10k(master, last_commit, git_environments_path)
```

Comment:
The indent on this line is a tab character versus spaces. Make sure you always use spaces because otherwise it will cause merge conflicts in the future. You can configure your editor to use spaces when you hit the tab key.